docs/methods/user.verifyEmail.md

```diff
@@ -0,0 +1,65 @@
+# user.verifyEmail
+
+`POST /api/user.verifyEmail`
+
+## Description
+
+Initiates the _email verification_ process for the designated user.
+
+***
+
+## Auth logic
+
+### A. Verifying the email address of another user
+
+Requestor must be authenticated and assigned with `yeep.user.write` permission in _global_ scope to initiate the email verification process of another user.
+
+> Bear in mind that a user authenticated with these permissions in the global scope can also set the `isVerified` property immediatelly on the user, bypassing the email verification process.
+
+### B. Verifying the requestor's own email address
+
+Users are able to request the verification of their own email address without explicit permissions.
+
+## Parameters
+
+### Body
+
+- **id** _(string)_ — the user id (required)
+- **emailAddress** _(string)_ — the user email address to verify (required)
+- **tokenExpiresInSeconds** _(number)_ — number of seconds after which the _email verification_ token will expire (optional; defaults to 10800 seconds, i.e. 3 hours)
```

Comment:
This param should only be set by superadmin users, otherwise a malicious user can create a token that lasts forever. Arguably it's a low security risk, but still a bad practice.